docs/en/contributor/Building-Alluxio-From-Source.md

```diff
@@ -17,22 +17,11 @@ This guide describes how to clone the Alluxio repository, compile the source cod
 - [Maven 3.3.9 or later](http://maven.apache.org/download.cgi)
 - [Git](https://git-scm.org/downloads)
 
-Alternatively, we have publishhed a docker image [alluxio/alluxio-maven](https://hub.docker.com/r/alluxio/alluxio-maven) with Java, Maven and Git installed to build Alluxio source code.
-For example, to build Alluxio with JDK8, checkout this image at tag `0.0.5-jdk8`:
+Alternatively, we have published a docker image [alluxio/alluxio-maven](https://hub.docker.com/r/alluxio/alluxio-maven) with Java, Maven, and Git pre-installed to help build Alluxio source code.
+To use this docker image, check it out locally first:
```

Comment:
oh i just realized that you don't need to explicitly pull an image; `docker run` will pull if the image doesn't exist locally. so we can omit this following section completely.